src/file.mock.ts

```diff
@@ -0,0 +1,20 @@
+export class MockFile {
```

Comment:
Move this to `file.spec.ts` as we do not use it anywhere else besides there.

Reply:
Will move it.